github/operations.py

```diff
@@ -583,6 +622,44 @@ def get_web_url(config, params, *args, **kwargs):
     return result
 
 
+def get_file_from_repository(config, params, *args, **kwargs):
+    github = GitHub(config)
+    payload = {'ref': params.get('branch') if params.get('branch') else 'main'}
+    endpoint = '{0}/contents/{1}'.format(params.get('name'), params.get('path'))
+    response = github.make_request(endpoint=endpoint, org=params.get('org'), owner=params.get('owner'), params=payload)
+    if params.get('decode_content') and response.get('content'):
+        try:
+            decoded_bytes = base64.b64decode(response.get('content'))
+            response['content'] = decoded_bytes.decode('utf-8')
+        except Exception as err:
+            logger.error(err)
+    return response
+
+
+def delete_file_from_repository(config, params, *args, **kwargs):
+    github = GitHub(config)
+    payload = {
```

Comment:
You can use same logic here:
payload = {k: v for k, v in params.items() if
               v is not None and v != '' and v != {} and v != []}